### ISSUE TITLE
ELLIPSIS in C

### ISSUE THREAD

**Issue by BenShen98** [LangProc/langproc-2018-cw#8]
Do we need to consider ELLIPSIS `...` in C? [ref](https://stackoverflow.com/questions/3792761/what-is-ellipsis-operator-in-c)

**Comment by ymherklotz** [comment]
I do not think that it has to be considered, as it is not in the list of features.
